test/Interop/Cxx/class/constructors-irgen.swift

```diff
@@ -0,0 +1,94 @@
+// Target-specific tests for C++ constructor call code generation.
+
+// RUN: %swift -module-name Swift -target x86_64-apple-macosx10.9 -dump-clang-diagnostics -I %S/Inputs -enable-cxx-interop -emit-ir %s -parse-stdlib -parse-as-library -disable-legacy-type-info | %FileCheck %s -check-prefix=ITANIUM_X64
+// RUN: %swift -module-name Swift -target armv7-none-linux-androideabi -dump-clang-diagnostics -I %S/Inputs -enable-cxx-interop -emit-ir %s -parse-stdlib -parse-as-library -disable-legacy-type-info | %FileCheck %s -check-prefix=ITANIUM_ARM
+// RUN: %swift -module-name Swift -target x86_64-unknown-windows-msvc -dump-clang-diagnostics -I %S/Inputs -enable-cxx-interop -emit-ir %s -parse-stdlib -parse-as-library -disable-legacy-type-info | %FileCheck %s -check-prefix=MICROSOFT_X64
+
+import Constructors
+import TypeClassification
+
+typealias Void = ()
+struct UnsafePointer<T> { }
+struct UnsafeMutablePointer<T> { }
+
+public func createHasVirtualBase() -> HasVirtualBase {
```

Comment:
Should I remove the below comments now that the patch no longer modifies any of the logic around applying attributes, or is the information still helpful?

Reply:
I'd drop them.

Reply:
SGTM.